### PR TITLE
[ZEPPELIN-4241] Configure the interpreter docker container time zone

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
Need to set the interpreter container and zeppelin service to have the same time zone
Otherwise, the time in the container is inconsistent with the zeppelin server time.

### What type of PR is it?
[Improvement]

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-4241

### How should this be tested?
[CI pass](https://travis-ci.org/liuxunorg/zeppelin/builds/566315011)

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

File: docs/quickstart/docker.md [hunk 1]
```diff
@@ -58,21 +58,28 @@ vi `/etc/docker/daemon.json`, Add `tcp://0.0.0.0:2375` to the `hosts` configurat
...
+  <description>Docker image for interpreters</description>
+</property>
+```
+
+2. set timezone in zeppelin-env.sh
 
+Set to the same time zone as the zeppelin server, keeping the time zone in the interpreter docker container the same as the server. E.g, `"America/New_York"` or `"Asia/Shanghai"`
+
+```
+export DOCKER_TIME_ZONE="America/New_York"
```
Comment (zjffdu) [inline docs/quickstart/docker.md]: Is there any default value?

File: docs/quickstart/docker.md [hunk 2]
```diff
@@ -90,12 +97,14 @@ ENV SPARK_VERSION=2.3.3
 ENV HADOOP_VERSION=2.7
 
 # support Kerberos certification
-RUN install -yq curl unzip wget grep sed vim krb5-user libpam-krb5 && apt-get clean
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -yq krb5-user libpam-krb5 && apt-get clean
```
Comment (zjffdu) [inline docs/quickstart/docker.md]: Why this change ？It seems not related with timezone

File: zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java [hunk 3]
```diff
@@ -451,10 +459,9 @@ private void copyRunFileToContainer(String containerId)
 
       // 8) ${ZEPPELIN_HOME}/lib/interpreter is uploaded to `${CONTAINER_ZEPPELIN_HOME}`
       //    directory in the container
-      String libIntpPath = "/lib/interpreter";
-      String zeplLibIntpPath = getPathByHome(zeppelinHome, libIntpPath);
-      mkdirInContainer(containerId, zeplLibIntpPath);
-      docker.copyToContainer(new File(zeplLibIntpPath).toPath(), containerId, zeplLibIntpPath);
+      String intpApiJarPath = "/interpreter/zeppelin-interpreter-api-0.9.0-SNAPSHOT.jar";
```
Comment (zjffdu) [inline zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java]: version is hard coded

File: docs/quickstart/docker.md (hunk 1 above)
Comment (liuxunorg) [reply docs/quickstart/docker.md]: This is the world's various time zones, a generic string, not our custom string.

File: docs/quickstart/docker.md (hunk 2 above)
Comment (liuxunorg) [reply docs/quickstart/docker.md]: When installing `krb5` in the dockerfile, Need to set `DEBIAN_FRONTEND=noninteractive`, Otherwise it will wait for user input.

This docker.md file is a documentation for zeppelin interpreter on docker, so it was updated.

File: zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java (hunk 3 above)
Comment (liuxunorg) [reply zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java]: Oh, this is a test code, Forgot to delete, wait i revert code.

**Comment by liuxunorg** [comment]
Deleted test code and [CI Pass](https://travis-ci.org/liuxunorg/zeppelin/builds/566412239)

File: docs/quickstart/docker.md [hunk 4]
```diff
@@ -192,8 +201,9 @@ Instead of build Zeppelin distribution package and docker image everytime during
 Zeppelin can run locally (such as inside your IDE in debug mode) and able to run Interpreter using [DockerInterpreterLauncher](https://github.com/apache/zeppelin/blob/master/zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterLauncher.java) by configuring following environment variables.
 
 
+1. zeppelin-site.xml
```
Comment (zjffdu) [inline docs/quickstart/docker.md]: zeppelin-env.sh ?

Comment (liuxunorg) [reply docs/quickstart/docker.md]: 1. `ZEPPELIN_RUN_MODE ` already exist in https://github.com/apache/zeppelin/blob/master/conf/zeppelin-site.xml.template#L617
2. `ZEPPELIN_DOCKER_CONTAINER_IMAGE ` already exist in https://github.com/apache/zeppelin/blob/master/conf/zeppelin-site.xml.template#L647

i modify `Environment variable` to `Configuration variable`.

**Comment by liuxunorg** [comment]
Modify `Environment` to `Configuration` and [CI Pass](https://travis-ci.org/liuxunorg/zeppelin/builds/566758681)

File: docs/quickstart/docker.md (hunk 2 above)
Comment (felixcheung) [reply docs/quickstart/docker.md]: it could help to explain that in comment

File: docs/quickstart/docker.md [hunk 5]
```diff
@@ -90,12 +97,14 @@ ENV SPARK_VERSION=2.3.3
 ENV HADOOP_VERSION=2.7
 
 # support Kerberos certification
-RUN install -yq curl unzip wget grep sed vim krb5-user libpam-krb5 && apt-get clean
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -yq krb5-user libpam-krb5 && apt-get clean
+
+RUN apt-get update && apt-get install -y curl unzip wget grep sed vim tzdata
```
Comment (felixcheung) [inline docs/quickstart/docker.md]: do this before the krb5 one?
also, at the end should add ` && apt-get clean`

Comment (liuxunorg) [reply docs/quickstart/docker.md]: 1. They don't have a sequence, I just want to separate the installation of these two components.
2. Ok, let me add `&& apt-get clean`

**Comment by liuxunorg** [comment]
Will merge if no more comments

**Comment by liuxunorg** [comment]
[CI Pass](https://travis-ci.org/liuxunorg/zeppelin/builds/567315497)
